infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -104,4 +134,8 @@ to build the images and start the `local.spark-master` container.
 
 ### Start spark-shell
 
-Once the terraform services have started on AKS, run `./spark-client.sh`. This script gets the AKS cluster info such storage account and AKS cluster using terraform. And then pulls the details of the services running on AKS such as tables-service  address and port and generates the spark-shell start command. Lastly, starts spark shell within the `local.spark-master` container. Then, run commands like those found in `SETUP.md` to make requests from the spark client.
\ No newline at end of file
+Once the terraform services have started on AKS, run `./spark-client.sh`. This script gets the AKS cluster info such storage account and AKS cluster using terraform. And then pulls the details of the services running on AKS such as tables-service  address and port and generates the spark-shell start command. Lastly, starts spark shell within the `local.spark-master` container. Then, run commands like those found in `SETUP.md` to make requests from the spark client.
+
+## Troubleshooting
+
+I have compiled a doc of common errors and fixes [here](https://docs.google.com/document/d/e/2PACX-1vQqnmJBPutMzinJ_ACv2dpI0m6X9HNkb4t7W_qYRU4ICuBrY8qXC_zsAVH0P_0RXEUeiDkXYhFFewPy/pub).
```

Comment:
is this doc public in your personal space? if not, please make it so

Reply:
Yes I will do that